cli-feed-v4.json

```diff
@@ -41,7 +41,7 @@
       "hidden": false
     },
     "v4": {
```

Comment:
We need a pre-release tag here. There wasn't one before because v4 was in preview.